### PR TITLE
JAVAFICATION: Port SecretStore to Java

### PR DESCRIPTION
**Pull request by danhermann**

### CODE REVIEW

File: logstash-core/src/main/java/org/logstash/secret/store/SecretStoreExt.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package org.logstash.secret.store;
+
+import org.jruby.RubyHash;
+import org.logstash.RubyUtil;
+import org.logstash.secret.SecretIdentifier;
+
+public class SecretStoreExt {
+
+    private static SecretStoreFactory SECRET_STORE_FACTORY = SecretStoreFactory.fromEnvironment();
```
Comment (original-brownbear) [inline logstash-core/src/main/java/org/logstash/secret/store/SecretStoreExt.java]: nit: Final

**Comment by original-brownbear** [comment]
@danhermann LGTM but there must be a detail off with the tests.

**Comment by danhermann** [comment]
Jenkins, test this please

**Comment by danhermann** [comment]
@original-brownbear, thanks! Tests are green now.

**Pull request closed without merging**